package.json

```diff
@@ -6,7 +6,8 @@
   "scripts": {
     "sort": "./bin/sort words.json > sorted.json && mv sorted.json words.json",
     "lint": "jsonlint -q words.json",
-    "test": "npm run lint && tape test.js"
+    "test": "npm run lint && tape test.js",
+    "validate": "npm ls"
```

Comment:
@fidelisrafael I just asked you a question. I'm not arguing against the change that you made, I actually like and appreciate it.